packages/dialtone-vue2/components/rich_text_editor/extensions/custom_link/index.js

```diff
@@ -0,0 +1,5 @@
+import { CustomLink } from './custom_link';
+
+export * from './custom_link';
```

Comment:
is there a reason to export everything?

Reply:
no don't think so... just copied what was there. Will remove.